.github/workflows/modules-test.yml

```diff
@@ -38,6 +38,34 @@ jobs:
       - name: cargo test --locked
         working-directory: ./modules
         run: cargo test --locked --all-features --all-targets
+  coverage:
```

Comment:
We can't run this here. The github runners are too light. We have a circleCI job running [here](https://github.com/AbstractSDK/abstract/blob/main/.circleci/config.yml#L119C7-L119C24) that does the coverage for the modules. 

So when uploading the file you don't have to specify anything related to the component to tell codecov which compontent you're uploading for? 

Specifically looking in this action: https://github.com/codecov/codecov-action

Don't know, just wondering

Reply:
I don't think we need to specify anything in particular to tell it which component to look at, the components are just used for the structure of the report I think. As in, when we run the coverage it will see which coverage comes from which folders and format accordingly. I could be wrong though, first time using this tool :)  Good catch on the circleci stuff, I will update this to just use that.

Reply:
Ahh okey! Thanks for looking into it and explaining :)

Reply:
Latest commit uncomments the commented out action which triggers circleci to run the codecov job. I am not sure why this was commented out but I think this should work.

Reply:
Because the coverage takes a long time to compute so it burns through our plan.